econml/iv/dr/_dr.py

```diff
@@ -554,32 +1937,35 @@ class IntentToTreatDRIV(_IntentToTreatDRIV):
     """
 
     def __init__(self, *,
-                 model_Y_X,
-                 model_T_XZ,
+                 model_y_xw="auto",
+                 model_t_xwz="auto",
                  flexible_model_effect,
                  model_final=None,
+                 z_propensity="auto",
```

Comment:
Similar to DRIV, I think here too we should enable a similar interface on the choice of preliminary model, i.e. add params
`prel_cate_estimator={'driv', 'dmliv'}`, `prel_cv=1`, `prel_opt_reweighted=True`.

If `prel_cate_estimator` is `'dmliv'`, we can instantiate a NonParamDMLIV as `prel_model_effect`. Only caveat is that NonParamDMLIV also expects `model_t_xw`, similar to what we do in the nuisance model, this is simply `model_t_xwz(X,W, 1) * dummy_z(X, W) + model_t_xwz(X,W,0) * (1 - dummy_z(X,W))`, so we can wrap these estimators into a `model_t_xw` and pass that to DMLIV.

Reply:
done!